terraform/cache/services/provider.go

```diff
@@ -0,0 +1,298 @@
+package services
+
+import (
+	"context"
+	goerrors "errors"
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/gruntwork-io/go-commons/errors"
+	"github.com/gruntwork-io/terragrunt/pkg/log"
+	"github.com/gruntwork-io/terragrunt/terraform/cache/models"
+	"github.com/gruntwork-io/terragrunt/util"
+	"github.com/hashicorp/go-getter/v2"
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform/command/cliconfig"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/sync/errgroup"
+)
+
+var (
```

Comment:
Looks like variables are used as constants, can it be declared as `const`?

Reply:
Ah, indeed, thanks. Fixed.